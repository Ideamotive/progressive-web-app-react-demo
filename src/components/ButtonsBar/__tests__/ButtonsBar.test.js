import React from 'react';
import * as renderer from 'react-test-renderer';

import ButtonBar from '../Bar';

describe('<ButtonBar/> component', () => {
  test('should match to snapshot', () => {
    const tree = renderer
      .create(<ButtonBar
        backgroundColor={'#fff'}
      >
        <button>Click</button>
      </ButtonBar>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

