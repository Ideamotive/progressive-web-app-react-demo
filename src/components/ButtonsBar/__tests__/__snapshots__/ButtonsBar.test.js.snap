// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonBar/> component should match to snapshot 1`] = `
<div
  className="sc-bdVaJa ciCIem"
>
  <button>
    Click
  </button>
</div>
`;
